gemrb/core/Map.cpp

```diff
@@ -2907,23 +2907,23 @@ void Map::BlockSearchMap(const Point &Pos, unsigned int size, unsigned int value
 				unsigned int ppypj = ppy+j;
 				unsigned int ppxmi = ppx-i;
 				unsigned int ppymj = ppy-j;
-				if ((ppxpi<Width) && (ppypj<Height)) {
-					unsigned int pos = ppypj*Width+ppxpi;
+				unsigned int pos = ppypj*Width+ppxpi;
```

Comment:
Please use the occasion to also fix the whitespace of the lines you edit. This our main way to harmonize style without polluting history.

Reply:
next time please only do **the lines you edit**, otherwise it's no different.